scalafmt-tests/src/test/resources/align/DefaultWithAlign.stat

```diff
@@ -2071,3 +2072,21 @@ object a {
   def meth3(param333333:     Any):    Any = ???
   def md4(param4:            Any):    Any = ???
 }
+<<< #3614 !spaces.inParentheses !align.delayUntilSpace
+spaces.inParentheses = false
+align.delayUntilSpace = false
+align.tokens."+" = [ { code = ":" }, { code = "(" }, { code = ")" }, { code = "=" } ]
+===
+object a {
+  def meeethod1(pram1: AnyRef): Any = ???
+  def methd2(paaaaaram2: Any): Any = ???
+  def meth3(param333333: Any): Any = ???
+  def md4(param4: Any): Any = ???
+}
+>>>
+object a {
```

Comment:
Actually, I would have thought it would work like this always with alignment, though I don't use so never tested. I guess since this changes the default, we need the additional flag.